packages/earn-controller/src/EarnController.ts

```diff
@@ -611,27 +555,32 @@ export class EarnController extends BaseController<
    * Refreshes pooled staking vault daily apys for the current chain.
    * Updates the pooled staking vault daily apys controller state.
    *
+   * @param chainId - The chain id to refresh pooled staking vault daily apys for (optional).
    * @param days - The number of days to fetch pooled staking vault daily apys for (defaults to 365).
    * @param order - The order in which to fetch pooled staking vault daily apys. Descending order fetches the latest N days (latest working backwards). Ascending order fetches the oldest N days (oldest working forwards) (defaults to 'desc').
    * @returns A promise that resolves when the pooled staking vault daily apys have been updated.
    */
   async refreshPooledStakingVaultDailyApys(
+    chainId: number = ChainId.ETHEREUM,
```

Comment:
This is a breaking change. In the future, perhaps consider grouping these arguments into an options bag so that a new property can be added to the options without affecting the method signature.

Reply:
Also a great callout. And it's more in line with how we call most of these other similar methods so I think I'll go ahead and do that.